.gitignore

```diff
@@ -5,8 +5,8 @@ mstpd
 
 /bridge-stp
 /bridge-stp.tmp
-/lib/ifupdown.sh
```

Comment:
ah ; i forgot to add ; if you do really want this change, there is always the democratic approach, where other members of the group could be persuaded otherwise ; 
so, if they are convinced by it, i am fine with it